src/test/java/io/github/vlsergey/springdatarestutils/EntityToSchemaMapperTest.java

```diff
@@ -30,10 +33,11 @@ class EntityToSchemaMapperTest {
 
     @Test
     void testLink() throws Exception {
+	Yaml yaml = new Yaml(new Constructor(Map.class));
```

Comment:
Can all the `Yaml yaml = new Yaml(new Constructor(Map.class));` be extracted out and shared?

Reply:
Done